### ISSUE TITLE
small memleak on RAI_ModelCreateTF

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#279]
```
23736:M 20 Feb 2020 01:33:30.651 # Redis is now ready to exit, bye bye...
==23736== 
==23736== HEAP SUMMARY:
==23736==     in use at exit: 10,966,091 bytes in 212,513 blocks
==23736==   total heap usage: 633,781 allocs, 421,268 frees, 51,604,154 bytes allocated
==23736== 
==23736== 4 bytes in 1 blocks are definitely lost in loss record 42 of 125,128
==23736==    at 0x4C2FB0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==23736==    by 0x144D38: zmalloc (zmalloc.c:99)
==23736==    by 0x144F83: zstrdup (zmalloc.c:210)
==23736==    by 0x1C7006: RM_Strdup (module.c:352)
==23736==    by 0x16EF2140: RAI_ModelCreateTF (tensorflow.c:312)
==23736==    by 0x67E69A7: RAI_ModelCreate (model.c:195)
==23736==    by 0x67D784B: RedisAI_ModelSet_RedisCommand (redisai.c:620)
==23736==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==23736==    by 0x13D315: call (server.c:2468)
==23736==    by 0x13DFCA: processCommand (server.c:2763)
==23736==    by 0x14F086: processInputBuffer (networking.c:1470)
==23736==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
==23736== 
==23736== LEAK SUMMARY:
==23736==    definitely lost: 4 bytes in 1 blocks
==23736==    indirectly lost: 0 bytes in 0 blocks
==23736==      possibly lost: 124,286 bytes in 1,457 blocks
==23736==    still reachable: 2,852,301 bytes in 65,302 blocks
==23736==                       of which reachable via heuristic:
==23736==                         stdstring          : 2,230,937 bytes in 56,721 blocks
==23736==                         newarray           : 3,424 bytes in 4 blocks
==23736==         suppressed: 7,989,500 bytes in 145,753 blocks
==23736== Reachable blocks (those to which a pointer was found) are not shown.
==23736== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==23736== 
==23736== For counts of detected and suppressed errors, rerun with: -v
==23736== ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 1 from 1)
	bad exit code for serverId 1
	Redis did not exit cleanly
basic_tests:test_example_multiproc
```

**Comment by lantiga** [comment]
Fixed in #289

**Comment by filipecosta90** [comment]
fixed :+1:
